From a7a9cea5471babcf1055f1552f0946dda8927c40 Mon Sep 17 00:00:00 2001
From: Omair Majid <omajid@redhat.com>
Date: Tue, 17 Nov 2020 13:14:19 -0500
Subject: [PATCH] Fix building in a source-build context

The GitVersionTask is not compatible with .NET Core. So disable it and
use an explicit PackageVersion.

The NuGet feed points to something that returns an error for me. So just
disable it for now.

Disable SourceLink.Create.CommandLine since upstream is dead and we dont
really need it for now.
---
 src/Humanizer/Humanizer.csproj | 5 ++---
 src/NuGet.config               | 5 ++++-
 2 files changed, 6 insertions(+), 4 deletions(-)

diff --git a/NuGet.config b/NuGet.config
new file mode 100644
index 0000000..43bd4ce
--- /dev/null
+++ b/NuGet.config
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+   <packageSources>    
+     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+   </packageSources>
+</configuration>
diff --git a/src/Humanizer/Humanizer.csproj b/src/Humanizer/Humanizer.csproj
index 5b2f8ad..51adb3c 100644
--- a/src/Humanizer/Humanizer.csproj
+++ b/src/Humanizer/Humanizer.csproj
@@ -3,6 +3,7 @@
     <TargetFrameworks>netstandard1.0;netstandard2.0;net6.0</TargetFrameworks>    
     <Authors>Mehdi Khalili, Claire Novotny</Authors>
     <PackageLicenseUrl>https://raw.githubusercontent.com/Humanizr/Humanizer/master/LICENSE</PackageLicenseUrl>
+    <PackageVersion>2.14.1</PackageVersion>
     <PackageProjectUrl>https://github.com/Humanizr/Humanizer</PackageProjectUrl>
     <MinClientVersion>2.12</MinClientVersion>
     <Description>A micro-framework that turns your normal strings, type names, enum fields, date fields ETC into a human friendly format</Description>    
